core/src/main/java/org/jsmart/zerocode/core/kafka/consume/ConsumerLocalConfigs.java

```diff
@@ -20,10 +20,13 @@
     private final Integer maxNoOfRetryPollsOrTimeouts;
     private final Long pollingTime;
     private final String seek;
+    private final String protobufMessageClassType;
+    
 
     @JsonCreator
     public ConsumerLocalConfigs(
             @JsonProperty("recordType") String recordType,
+            @JsonProperty("protobufMessageClassType") String protobufMessageClassType,
```

Comment:
Update : `protobufMessageClassType` -> `protoClassType` please! Meaningful n less to remember :-)

Appreciate if you could take care of this.

Reply:
agree will change thanks